cgroup/cgroup.go

```diff
@@ -0,0 +1,77 @@
+package cgroup
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/godbus/dbus/v5"
+	"github.com/godbus/dbus/v5/introspect"
+	"github.com/godbus/dbus/v5/prop"
+
+	logging "github.com/home-assistant/os-agent/utils/log"
+)
+
+const (
+	objectPath            = "/io/homeassistant/os/CGroup"
+	ifaceName             = "io.homeassistant.os.CGroup"
+	CGroupFSDockerDevices = "/sys/fs/cgroup/devices/docker/"
```

Comment:
Since its only used locally this should be small caps
```suggestion
	cGroupFSDockerDevices = "/sys/fs/cgroup/devices/docker/"
```